### ISSUE TITLE
Shuffle the training and test images

### ISSUE THREAD

**Issue by DavidZong** [DavidZong/transfer_learning_project#9]
The test set is just the first 30% of the whole set, but that makes some of the test images highly correlated. Best to shuffle them into the training set and re-slice.
